cmd/secrets/read_secret.go

```diff
@@ -0,0 +1,145 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+// +build secrets
```

Comment:
We don't need to hide this behind the secrets build flag anymore, no ?

Reply:
Not as of right now, but we'd still need to if we reuse the consts from the `secret` package.

Reply:
indeed :+1: